Automattic/wp-calypso
# Checkout: Fix long domain names breaking checkout modal

Related to https://github.com/Automattic/wp-calypso/issues/95110
Fixes https://github.com/Automattic/wp-calypso/issues/95110

## Proposed Changes

* Long domain names are breaking the modal when removing a domain from the cart.
* This PR adds CSS changes to break the long domain names into new lines

## Why are these changes being made?


* Fix layout issues caused by long domain names

## Testing Instructions



* Use live link
*  Using any flow when reaching the domains step, choose a huge domain name
* Proceed further until checkout
* Try to remove the domain from the cart to see the modal
* The long domain name should not look bad in the modal 

## Pre-merge Checklist
